pages/data-providers/[data-provider-id]/content.jsx

```diff
@@ -61,12 +61,14 @@ const tableConfig = {
       expandedSize: 1,
       className: styles.visibilityColumn,
       getter: v => !v.disabled,
-      render: (v, isExpanded) => (
+      render: (v, { id, isExpanded }) => (
         <PublishedToggle
           className={styles.visibilitySwitch}
           defaultVisibility={v}
           isExpanded={isExpanded}
-          disabled
+          onVisibilityChanged={isVisible =>
```

Comment:
I think that if you have a wrapper around checkbox it should completely implement checkbox interface at least because checkbox interface is a standard.